casper/src/main/resources/PoS.rhox

```diff
@@ -65,7 +70,7 @@ new  PoS,
             contract PoS (@"getRewards", returnCh) = {
               new tmpCh in {
                 getMVar!(*stateCh, *tmpCh) |
-                for (@(rewards, _) <- tmpCh) {
+                for (@(_, rewards, _) <- tmpCh) {
```

Comment:
Please document that `getRewards` returns the `pending + committed` rewards that a validator had _before_ the last block was closed.